fecgov/openFEC
# Increase size of tres_nm column in f_rpt_or_form_sub table

## Summary (required)

- Resolves #4463 

As part of deploying the Informatica mappings to populate the tres_nm field 
in f_rpt_or_form_sub table from tres_sign_nm field in nml form tables, 
the size of tres_nm field in f_rpt_or_form_sub table 
from varchar2(38) to varchar2(90) in FECP database. 
The size of tres_nm column in f_rpt_or_form_sub table in Postgress databases 
also need to be increased to accommodate this change.

Currently there are MVs/VWs referencing f_rpt_or_form_sub table. 
Two of them referencing this particular column 
so these MVs/VWs need to be updated as well.

## How to test the changes locally
- Download the branch to local server, run flyway migration or run invoke create_sample_db, make sure migration files executed successfully
- Run pytest.
- log into local cfdm_test database, check the size of tres_nm column in f_rpt_or_form_sub table, it should be varchar(90)
- check ddl of ofec_totals_combined_mv, one of the source of this mv should be ofec_filings_all_vw, not ofec_filings_vw
- AFTER the migration files had executed, run the following statement in cfdm_test database
`delete from flyway_schema_history where version in ('0203', '0204', '0205');`
Then run flyway migrate again, make sure these migration files got executed again without error message.  This is to test the condition when these migration files merged into the AWS databases since these will be implement in the AWS databases already.

## Impacted areas of the application
List general components of the application that this PR will affect:

-  No API impacted.



## Related PRs
List related PRs against other branches:

branch | PR
------ | ------
fix/other_pr | [link]()
feature/other_pr | [link]()
